R/fetch_survey.R

```diff
@@ -67,6 +67,11 @@
 #'   details. Defaults to `NULL`. Overwritten by `convert = TRUE`.
 #' @param verbose Logical. If `TRUE`, verbose messages will be printed to the R
 #'   console. Defaults to `TRUE`.
+#' @param tmp_dir Path to filesystem directory. Qualtrics returns response data
+#'   in compressed (zip) form. To extract raw data, the zip file must be briefly
+#'   written to disk (the file is then promptly deleted). By default, the
+#'   system's temporary directory is used for this (see \code{tempdir()}), but
```

Comment:
```suggestion
#'   system's temporary directory is used for this (see [tempdir()], but
```

Can we link to this?

Reply:
We should.  And if I hadn't misremembered the proper syntax for how to do that, we already would have 👍.  Appreciate the fix.

Rejecting these specific amendments because I'm in an additional commit that also rebuilds the docs.  Thanks again.